frontend/test/acceptance/util/Browser.scala

```diff
@@ -55,6 +77,32 @@ trait Browser extends WebBrowser {
       throw new MissingPageElementException(q)
   }
 
+  // Switches to a new iframe specified by the Query, q.
+  def switchFrame(q: Query) {
+    if (pageHasElement(q))
+      driver.switchTo().frame(q.webElement)
+    else
+      throw new MissingPageElementException(q)
+  }
+
+  /*
+   * Switches to the first window in the list of windows that doesn't match
+   * the parent window.
+   * */
+  def switchWindow() {
```

Comment:
Do `switchWindow`, `switchToParentWindow`, and `changeUrl` fit better in `acceptance.util.Driver` instead of `Browser`?